pyproject.toml

```diff
@@ -134,7 +134,14 @@ integration = 'pytest --maxfail=5 -m "integration" test'
 integration-mac = 'pytest --maxfail=5 -m "integration" test -k "not tika"'
 integration-windows = 'pytest --maxfail=5 -m "integration" test -k "not tika"'
 types = "mypy --install-types --non-interactive --cache-dir=.mypy_cache/ {args:haystack}"
-lint = "pylint -ry -j 0 {args:haystack}"
+lint = [
+  "ruff {args:haystack}",
+  "pylint -ry -j 0 {args:haystack}"
+]
+lint-fix = [
+  "black .",
+  "ruff {args:haystack} --fix",
```

Comment:
We'd have to exclude every rule apart from the import sorting ones. Otherwise we'll have tons single line ignores.

Let's handle that in another PR maybe?

Reply:
👍🏻